src/main/scala/io/gatling/build/GatlingOssPlugin.scala

```diff
@@ -16,8 +16,7 @@
 
 package io.gatling.build
 
-import io.gatling.build.automated.GatlingAutomatedScalafixPlugin
-import io.gatling.build.automated.GatlingAutomatedScalafmtPlugin
+import io.gatling.build.automated.{GatlingAutomatedScalafixPlugin, GatlingAutomatedScalafmtPlugin}
```

Comment:
Application of the "groupedImports = Merge" clause

otherwise the plugin wanted to explode every import...
Maybe we should do it (java style?) to limit impact (and possible conflict) on git diff?

Reply:
Arf! outdated because I forgot to compile (and trigger scalafmt after scalafix)